test/cases/js-hash/expected/style.44c5db65e2b053b76abc.fde1566f2c635787864b.css

```diff
@@ -0,0 +1 @@
+.wX52cuPepLZcpDx5S3yYO { background: red; }
```

Comment:
The double hashes in the filename look scary 🤔 

I assume that actual users will want to use `[contenthash]` only instead of both `[contenthash]` and `[chunkhash]`

Reply:
Yes of course, this is just a test which tests both